arch/arm64/src/common/arm64_vectors.S

```diff
@@ -95,8 +82,6 @@ SECTION_FUNC(text, up_saveusercontext)
 #endif
     stp    x4,  x5,  [x0, #8 * REG_ELR]
 
-    arm64_exception_context_save x4 x5 x0
```

Comment:
Should you still save sp_el0 and exception depth here ? At least assert() uses up_saveusercontext() although in flat mode sp_el0 is not used.

Reply:
I prefer remove this, up_saveusercontext() is only for crash dump, and it is nothing useful for the dump, and it make the backtrace failed

Reply:
Fine by me